[BUG] TypeError: core.warn is not a function

**Describe the bug**
if commit failed, throw error `[error]TypeError: core.warn is not a function`

**To Reproduce**

**Expected behavior**
retry commit.

**Screenshots**
```
Run vn7n24fzkq/github-profile-summary-cards@release
Start...
Remove old cards...
Creating ProfileDetailsCard...
Creating ReposPerLanguageCard...
Creating CommitsPerLanguageCard...
Creating StatsCard...
Creating preview markdown...
Commit file...
##[error]TypeError: core.warn is not a function
##[error]core.warn is not a function
```

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
`core.warning()` is correct.
https://github.com/actions/toolkit/blob/%40actions/github%401.1.0/packages/core/src/core.ts#L130

--- a/src/app.js
+++ b/src/app.js
@@ -106,7 +106,7 @@ const main = async () => {
           if (retry == maxRetry) {
             throw error;
           }
-          core.warn(`Commit failed. Retry...`);
+          core.warning(`Commit failed. Retry...`);
         }
       }
     }

